src/hyp3_autorift/process.py

```diff
@@ -505,6 +519,8 @@ def main():
     )
     parser.add_argument('--bucket', help='AWS bucket to upload product files to')
     parser.add_argument('--bucket-prefix', default='', help='AWS prefix (location in bucket) to add to product files')
+    parser.add_argument('--esa-username', default=None, help="Username for ESA\'s Copernicus Data Space Ecosystem")
+    parser.add_argument('--esa-password', default=None, help="Password for ESA\'s Copernicus Data Space Ecosystem")
```

Comment:
Do we need to escape the single quota in `ESA's` since it's wrapped in double quotes?

Reply:
Done